pages/profile.tsx

```diff
@@ -1,37 +1,63 @@
 import React from "react";
-import { Container, ProgressBar } from "react-bootstrap";
+import { Button, Container, ProgressBar, Image } from "react-bootstrap";
+import Link from "next/link";
+import { FaExpand } from "react-icons/fa";
+import { useUser } from "../lib/hooks/auth";
 
 export default function Profile() {
+  const { user } = useUser();
   return (
-    <Container className="mt-5 text-center d-grid gap-2">
-      <h1>Statistics</h1>
-      <table className="Statistics">
-        <thead>
-          <tr>
-            <th scope="col">12</th>
-            <th scope="col">92</th>
-            <th scope="col">6</th>
-            <th scope="col">6</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td scope="row">Played</td>
-            <td>Win %</td>
-            <td>Current Streak</td>
-            <td>Maximum Streak</td>
-          </tr>
-        </tbody>
-      </table>
-      <Container className="mt-5 text-center d-grid gap-2">
-        <h1> Guess Distribution </h1>
-        <ProgressBar now={100 / 4} label={`1`} variant="success" />
-        <ProgressBar now={0} label={`0`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 1} label={`4`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-      </Container>
-    </Container>
+    <div className="container">
+      <div className="row">
+        <div className="col-sm-6 text-center">
+          <br />
+          <br />
+          <br />
+          <br />
+          <Image
+            src={user?.profilePicture}
+            alt="Profile picture"
+            width="200"
+            height="200"
+            roundedCircle
+          />
+          <br />
+          <br />
+          <h4>
+            {user?.givenName} {user?.familyName}
+          </h4>
+          <br />
+          <br />
+          <Link href="/game">
+            <Button variant="secondary" size="lg" className="">
+              Real Word of the Day <FaExpand />
```

Comment:
On my screen this button as well as my name appears squished on the screen, as if the column that this is contained within is much thinner than it should be. Consider rethinking the layout of the columns, it seems like you are using a lot of containers and columns inside of one another in a way that might not be the most optimal.